analysis/src/Hint.ml

```diff
@@ -82,7 +82,7 @@ let inlay ~path ~pos ~maxLength ~debug =
     Ast_iterator.default_iterator.value_binding iterator vb
   in
   let iterator = {Ast_iterator.default_iterator with value_binding} in
-  (if Files.exists path && Filename.check_suffix path ".res" then
+  (if Files.isResFile path then
```

Comment:
I'm not sure that this does it.
The temptation could be `else ...` for `.resi` files which would leak to a crash.
Any other ideas?

Reply:
Check the latest commit, what do you think about that? Slightly harder at to accidentally do the wrong thing at least I think, but still not fool proof.